### PR TITLE
[BEAM-1964] Fix lint issues for linter upgrade -2

### PR DESCRIPTION
**Pull request by sb2nov**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

R: @aaltay PTAL

### CODE REVIEW

File: sdks/python/apache_beam/io/textio.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def split_points_unclaimed(stop_position):
         # followed by a new line character. Since such a record is at the last
         # position of a file, it should not be a part of the considered range.
         # We do this check to ignore such records.
-        if len(record) == 0 and num_bytes_to_next_record < 0:
+        if not record and num_bytes_to_next_record < 0:
```
Comment (sb2nov) [inline sdks/python/apache_beam/io/textio.py]: will revert

File: sdks/python/apache_beam/io/filebasedsource_test.py [hunk 2]
```diff
@@ -118,7 +118,7 @@ def write_prepared_pattern(data, suffixes=None):
   if suffixes is None:
     suffixes = [''] * len(data)
   temp_dir = tempfile.mkdtemp()
-  assert len(data) > 0
+  assert data, 'Data (%s) seems to be empty' % data
```
Comment (sb2nov) [inline sdks/python/apache_beam/io/filebasedsource_test.py]: Moving this.

File: sdks/python/apache_beam/io/textio.py (hunk 1 above)
Comment (sb2nov) [inline sdks/python/apache_beam/io/textio.py]: I checked the others and seem okay.

**Comment by aaltay** [comment]
LGTM

**Comment by sb2nov** [comment]
Thanks

**Pull request closed without merging**